File: Backend/app/Support/AirQualityThresholds.php
<?php

namespace App\Support;

/**
 * Open-Meteo returns US AQI (0–500), not a 1–5 European-style index.
 */
final class AirQualityThresholds
{
    /** Unhealthy for Sensitive Groups and above. */
    public const int PoorUsAqi = 100;

    public static function isPoorUsAqi(?int $aqi): bool
    {
        return $aqi !== null && $aqi >= self::PoorUsAqi;
    }
}

File: Mobile/lib/shared/weather/air_quality_thresholds.dart
/// Shared US AQI helpers (Open-Meteo `us_aqi` is 0–500, not a 1–5 European index).
abstract final class AirQualityThresholds {
  /// Unhealthy for Sensitive Groups and above — only then warn for outdoor activity.
  static const int poorUsAqi = 100;

  static bool isPoorUsAqi(int? aqi) => aqi != null && aqi >= poorUsAqi;
}
